### DISCUSSION TITLE
Cheats Request and submission

### DISCUSSION THREAD

**Discussion by kmeps4** [GoldHEN/GoldHEN_Cheat_Repository#42] [Announcements]
Hello there:
1) On GoldHEN side we do not create/develop cheats... if you need to ask an exclusive cheat, please redirect to: https://twitter.com/PS4Trainer and join their Discord Server and ask for needed Cheat.
PS4Trainer Discord: https://discord.com/invite/bUEQj9p

If you will like to join GoldHEN server: https://discord.gg/pR5NTEVBGt

Have a great one!!!

**Comment by beatd91** [comment]
Could we please get a new discord link, I just get a notice saying invitation is expired.

**Reply by kmeps4** [reply]
Hello.
Done :)

**Reply by francescogelosi10-jpg** [reply]
Ho bisogno ho installato il file zip però dopo cosa devo fare

**Comment by Ahmunet** [comment]
Hello, I don't really have a request, just that 
I've remarked that when install CODMW 2019 CUSA08829 V0101
on goldhen chts it says V0102 even though its actually v0101. Any fix?

**Reply by kmeps4** [reply]
can you check on game information.. inside there "Update history please" that way you will know why it's version 01.02

**Comment by musidrili** [comment]
Hi.  I can't find any json cheats for cusa08933 Skyrim Vr 1.04 game.  If you can prepare a json file for this game, I would appreciate it.

**Comment by Ahmunet** [comment]
its says version 01.02 for all the cusa even the retail one.
But I was wondering wouldn't it possible to update the cheats to CUSA08829
v01.63
as that the newest version.
and its COD modern warfare 2019 not skyrim

**Reply by musidrili** [reply]
I was looking for cheats for cusa08933 skyrim vr, not for Call of duty modern warfare.  I saw that there is version 1.00 in the update history.  I have installed the latest 1.04 update.  Unfortunately there is no json file for this game.

**Comment by snakeout** [comment]
Hello, resident evil village CUSA18017 – USA (v1.12) CUSA18008​ – EUR​ (v1.11) does not work Same for Jump force CUSA11625 – USA (v3.01), can someone fix the problem?

**Comment by mihasjjj** [comment]
добрый день... можете сделать чит для titanfall-2 cusa04013 v1.07 спасибо!!!

**Comment by toesntickles** [comment]
Hi,
Not sure if this is the right place to ask but Just Cause 3 cheats v1.05 within the GoldHen cheats manager don’t seem to work. The ps4 trainer works so the cheats DO work, just not through GoldHen cheats manager. Any ideas why?

**Comment by amdmkhllsh** [comment]
Will the Spiderman Miles Morales V1.14 cheat be updated? Will Jurassic World Evolution V1.34 have cheats like Jurassic World 2?

**Comment by Raj2k01** [comment]
Hi, can you please add cheats for Snowrunner CUSA17425 – USA (v1.45)

**Comment by twilightprinceXV** [comment]
Request for Final Fantasy XV Comrades (CUSA13608) 1.03.... thanks in advance

**Comment by magico1981** [comment]
Hi, can you please add cheats for Dead Cells CUSA11253 V1.47

**Comment by jramos00741** [comment]
Hi, can you please add cheats for: 
Resident Evil Director`s Cut Dual Shock (SLUS00747 V1.00)
Resident Evil 2 Dual Shock (SLUS00748 V1.00) - Cheats on Goldhen does not work.
Resident Evil 3 Nemesis (SLUS00923 V1.00) - Cheats on Goldhen does not work

**Comment by Dre023** [comment]
Hi, can u add cheats for CUSA32137 Like A Dragon Infinite Wealth V1.16 please? Thank you

**Comment by ReelModZz** [comment]
HI just want to say excellent work on the hard work to all the trainers getting made,
would like to see if possible 
Alien Dark Descent CUSA34370 v01.08 FW 9.00
Stray CUSA33346 v01.02 FW9.00
again awsome work keep it up.

**Comment by Justin-Kredible** [comment]
Hello, not sure if this is still where request are made but here goes. Can you fix the health code for:
ELDEST SOULS (CUSA28529) Game crashes when you activate it
Street Fighter 6 (CUSA31498) Money works... but not the inf. health

and add cheats for:
Record of Lodoss War Deedlit in Wonder Labyrinth (CUSA29366)
Panzer Dragoon Remake (CUSA19484)
FIGHTN RAGE (CUSA17844) 
Scott Pilgrim vs the world (CUSA20085)
Odin Sphere Leifthrasir (CUSA05083)_

**Comment by chstrhn** [comment]
Hi.. can we please an updated version for Granblue fantasy relink cusa34766 version 1.30.. i am already 700 hours in game but still cant beat the very final boss not even once

**Comment by MrFunny8** [comment]
Tony Hawk Pro Skater 1+2 (CUSA17922 or CUSA20464)
Crash Team Racing Nitro Fueled (CUSA14876 or CUSA13795)
Destroy All Humans! 2 Reprobed (CUSA36164 or CUSA36167)
Destroy All Humans Clone Carnage CUSA32947 v1.03
LA Noire (CUSA09084 or CUSA09172)
LA Noire The VR Case Files CUSA15490 v1.02

**Comment by Jeed24** [comment]
Hello please I have noticed you have updated the cheats 5 days ago and still I can’t find lies of p version 1.05 cheat please do so to update it

**Comment by MrFunny8** [comment]
Please add game follow

Tony Hawk Pro Skater 1+2 (CUSA17922 or CUSA20464)
Crash Team Racing Nitro Fueled (CUSA14876 or CUSA13795)
Destroy All Humans! 2 Reprobed (CUSA36164 or CUSA36167)
Destroy All Humans Clone Carnage CUSA32947 v1.03
LA Noire (CUSA09084 or CUSA09172)
LA Noire The VR Case Files CUSA15490 v1.02

**Comment by Jeed24** [comment]
Hi you see I have lies of p version 1.05 and I can’t use goldhen cheat
because the ones there is for version 1.01 and 1.03 I need 1.05

**Comment by Dre023** [comment]
Again, please

Can u add cheats for CUSA32137 Like A Dragon Infinite Wealth V1.16 please?

Thank you

**Reply by darkgoliat** [reply]
were they ever added? They do show, but it always makes the game freeze. at least on my console..... 
From what Ive gathered by reading comments the v1.13 work. Too late for me though Im already more than 25h in....

**Comment by burnok26** [comment]
good day! can i request cheats for the witcher 3 CUSA 05772 1,0.

**Comment by awankurosagi** [comment]
Hello.. umm, can u build cheat for Harvest Moon The winds of Anthos CUSA42678 for version 01.02?? thank you... :)

**Comment by ruhumuz** [comment]
mrb iyi günler red dead redemption 2 1.32 hilesi nezman gelir ?

**Comment by Chokenz1007** [comment]
Link on ps4 trainer discord is expired.. I just get a notice saying invitation is expired.

**Comment by roccotano90** [comment]
Can you please make infinite money and tokens cheats for ride 2 EUR version (CUSA05921 v01.09), thank you

**Comment by zaxzax169** [comment]
Hello Can you please make cheat for disgaea 7  (CUSACUSA40840 – USA v01.05),
Please I would like to thank you very much if my request is granted

**Comment by Ahmunet** [comment]
Can we please if possible have cheats for COD black ops 6.

**Comment by katsuu16** [comment]
Hello. I would like to make a request. A cheat for PS4 11.00 Assassin's Creed Origins CUSA05625 V1.00. Thank you in advance.

**Comment by lordtinem** [comment]
hello 
can you please add Red Dead Redemption 2 1.32 ( CUSA08519)
thank you

**Comment by Garuda87** [comment]
Can you add batman arkham knight cusa00135 v1.18

**Comment by Altroo** [comment]
Hello, can you please add cheats for Elden ring CUSA18723 v01.22.
There are cheats for previous version, i just don't know how to port them to the newest version.

**Comment by Ahmunet** [comment]
Could you please make Call of duty black ops 6 cheats?

**Comment by awankurosagi** [comment]
awww, we not read the instruction eh?, but so am I... lol

its fun, but i hope u all can read again..

**Comment by therealjmaxximus** [comment]
The cheats for Shinobi Art of Vengeance do not work. The infinite health and the invincible cheat crash the game. The other cheats just don't work. 

Shinobi Art of Vengeance ver. 1.01

**Comment by BankaiCellor** [comment]
Dragon Ball Xenoverse 2 CUSA05350 v1.48 please, i think that is last update for this game

**Comment by icyglacial** [comment]
CUSA18723_01.22.shn (Elden Ring)
cheats not working, cause crashes, tested on 9.0fw

**Comment by lordtinem** [comment]
dying light the following **CUSA03946** up 1.37 plz
and thanks

**Comment by mateenuuat-wq** [comment]
Hello, can you please add cheats for Fantasy Life i The Girl Who Steals Time Digital Deluxe Edition 1.50 ( CUSA52673 )
thank you

**Comment by adixman12** [comment]
Please add Dying Light 2 Stay Human (CUSA12555) version 1.62. Thanks.

**Comment by Awalsh053** [comment]
please add cheats for final fantasy Vll remake CUSA07187 V1.03 Thanks for all you do

**Comment by aiskwad** [comment]
Hi. Please add GTA 5 CUSA00411 1.38 or 1.50
I really need it)) Thanks in advance)

**Comment by Nathanpcs** [comment]
Would it be possible for you to add the cheat for Assassin's Creed Black Flag to the American version?

CUSA00010_01.04

Thanks for all you do

**Comment by Nedjmo16** [comment]
A cheat menu for Deadpool cusa 03245 v01.01

**Comment by yade1472** [comment]
Please can you mod codm for ios. Im buying Please dm

**Comment by kwekuf577-hue** [comment]
Please can you make cheat for Battlefield V (CUSA08670 - EUR v1.26) I would be glad if my request is granted
